play/js/main_sandbox.js

```diff
@@ -162,7 +162,8 @@ function main(config){
 			{name:"Borda", voter:RankedVoter, election:Election.borda, margin:4},
 			{name:"Condorcet", voter:RankedVoter, election:Election.condorcet},
 			{name:"Approval", voter:ApprovalVoter, election:Election.approval, margin:4},
-			{name:"Score", voter:ScoreVoter, election:Election.score}
+			{name:"Score", voter:ScoreVoter, election:Election.score},
+			{name:"STAR", voter:ScoreVoter, election:Election.star, margin:4}
```

Comment:
I added margin:4 here to follow the pattern, but it didn't seem to do anything as far as I could tell.